internal/pkg/mapper/sharedcode/helper/helper.go

```diff
@@ -77,34 +77,23 @@ func (h *SharedCodeHelper) CheckTargetComponent(sharedCodeConfig *model.Config,
 	return nil
 }
 
-func (h *SharedCodeHelper) GetSharedCodeByPath(branchKey model.BranchKey, path string) (*model.ConfigState, error) {
-	// Get branch
-	branch, found := h.state.Get(branchKey)
-	if !found {
-		return nil, fmt.Errorf(`missing %s`, branchKey.Desc())
-	}
-
+func (h *SharedCodeHelper) GetSharedCodeByPath(parentPath, codePath string) (*model.ConfigState, error) {
 	// Get key by path
-	path = filesystem.Join(branch.Path(), path)
-	configStateRaw, found := h.state.GetByPath(path)
+	codePath = filesystem.Join(parentPath, codePath)
+	configStateRaw, found := h.state.GetByPath(codePath)
```

Comment:
Namiesto branchKey sa pouziva `parentPath`.
Projektu je to jedno, je to iba ina alternativa,.... a pouzitie sablony to fixuje.